formula-test/src/main/java/com/instacart/formula/test/TestFormula.kt

```diff
@@ -74,7 +74,7 @@ abstract class TestFormula<Input, Output> :
         return Evaluation(
             output = state.output,
             updates = context.updates {
-                events(Stream.onTerminate()) {
+                Stream.onTerminate().events {
```

Comment:
I wonder if "onEvent" would be something that is closer to what's the expected naming for this would be ?